src/rust/jsg/ffi.h

```diff
@@ -9,7 +9,7 @@
 
 // Forward declarations needed by v8.rs.h
 namespace workerd::rust::jsg {
-using Isolate = v8::Isolate;
+using IsolatePtr = v8::Isolate;
```

Comment:
Non-blocking but... This feels... off... especially down below where we have `IsolatePtr*` ... which is Isolate pointer pointer?? Not sure what useful purpose the `using IsolatePtr ...` is actually serving.